e2e_tests/conftest.py

```diff
@@ -44,6 +45,8 @@ def environment():
         api_token=admin_token,
     )
 
+    time.sleep(10)
```

Comment:
is it really necessary? :(

10s wait in almost(?) every test is a lot...

Reply:
huh? Not in every. Only once or twice as it's inside of session-wide pytest fixture

Reply:
Right, I've missed that `scope="session"`. Sorry.